src/main.py

```diff
@@ -1,74 +1,10 @@
 #!/usr/bin/env python
-
-import pandas as pd
-
-from job_titles_feature_extraction import get_stemmed_sentences
-from job_description_feature_extraction import (get_one_hot_encoded_words,
-                                                get_rake_keywords,
-                                                get_top_idf_features)
-from cleaning_functions import (get_one_hot_encoded_feature,
-                                get_binary_encoded_feature,
-                                update_location)
-
-train_raw_data_csv_file_name = '../data/Train_rev1.csv'
-train_normalized_location_file_name = '../data/train_normalised_location.csv'
-test_raw_data_csv_file_name = '../data/Test_rev1.csv'
+from models.OldMainMethod import OldMainMethod
```

Comment:
i didn't want delete somebody's code
long term we should carefully move some parts to the base and some parts to a specific model - i can give it a start, but don't want to be the only one to decide that

it will increase the complexity of this PR but i can do it (questionmark?)